tb_pulumi/ec2.py

```diff
@@ -133,7 +133,7 @@ def __init__(
             target_type='ip',
             vpc_id=subnets[0].vpc_id,
             tags=self.tags,
-            opts=pulumi.ResourceOptions(parent=self, depends_on=[nlb]),
+            opts=pulumi.ResourceOptions(parent=self, depends_on=[nlb, subnets[0]]),
```

Comment:
I was going to ask what was so important about the first subnet, but the docstring answers that question! Should we maybe pop that into a variable?